iron-form.d.ts

```diff
@@ -34,12 +34,21 @@
  * want to submit it from a custom element's click handler, you need to explicitly
  * call the `iron-form`'s `submit` method.
  *
- *   Example:
+ *   Example (using `<paper-button>` for the form `<iron-form id="myForm">`):
+ *
+ *     <paper-button raised on-click="submitForm">Submit</paper-button>
+ *
+ *     function submitForm() {
+ *       this.$.myForm.submit();
+ *     }
+ *
+ *   or (using `<button>` for the same form):
  *
- *     <paper-button raised onclick="submitForm()">Submit</paper-button>
+ *     <button raised onclick="submitForm()">Submit</button>
  *
  *     function submitForm() {
- *       document.getElementById('iron-form').submit();
+ *       // `this.$.myForm.submit();`  or
```

Comment:
Argh, actually, really sorry I'm a pain in the ass. This doesn't make a ton of sense as a comment (if you have `this.$.myForm`, that means you're in a custom element and you can't also do `document.getElementById`. Would you mind deleting this comment?
(and maybe changing "using `<button>` for the same form" to "using `<button>` for a form in the same document")

Really really sorry I didn't catch it early.

Reply:
Certainly. I'll drop the line and regenerate the `*.d.ts`. I appreciate your help. But before I do (and commit again), if you want to spare folks ugly commits, would you explain _in painful detail_ what I do to "squash commits"? I've read [the directions](http://blog.steveklabnik.com/posts/2012-11-08-how-to-squash-commits-in-a-github-pull-request), but I'm not sure I understand (I'm only Git person here; the other dinosaurs write C++ and use CVS). (I'll set `$EDITOR` if that helps.)

Reply:
I can squash it in the UI before I merge, don't worry about it!

Reply:
(otherwise, if you're just curious, [this](http://blog.steveklabnik.com/posts/2012-11-08-how-to-squash-commits-in-a-github-pull-request) is what i do. but again, you don't need to do this, i can squash it for you)

((and super sorry again))